admin_ui/src/components/DurationWidget.vue

```diff
@@ -88,11 +90,11 @@ export default {
             hourRange: [...Array(24).keys()],
             minuteRange: [...Array(60).keys()],
             secondRange: [...Array(60).keys()],
-            weeks: undefined,
-            days: undefined,
-            hours: undefined,
-            minutes: undefined,
-            seconds: undefined,
+            weeks: undefined as any,
+            days: undefined as any,
+            hours: undefined as any,
+            minutes: undefined as any,
+            seconds: undefined as any,
```

Comment:
Instead of `undefined as any`, does `undefined as Number | undefined` work?

Reply:
No. If we don't want to use `undefined as any` only setting default to 0 works
```javascript
weeks: 0,
days: 0,
hours: 0,
minutes: 0,
seconds: 0,
```
I can change this if you think it's better.

Reply:
Yes, good idea - I think 0 is cleaner.